examples/objects_v2/deploy_objects_v2.yml

```diff
@@ -0,0 +1,141 @@
+        cluster_ext_id: "{{ cluster.uuid }}"
+        description: "object store test"
+        domain: "{{ objects.domain }}"
+        num_worker_nodes: "{{ objects.num_worker_nodes }}"
+        storage_network_reference: "{{ objects.storage_network_reference.uuid }}"
+        storage_network_vip:
+          ipv4:
+            value: "{{ objects.storage_network_reference.IPs[0] }}"
+        storage_network_dns_ip:
+          ipv4:
+            value: "{{ objects.storage_network_reference.IPs[1] }}"
+        public_network_reference: "{{ objects.public_network_reference.uuid }}"
+        public_network_ips:
+          - ipv4:
+              value: "{{ objects.public_network_reference.IPs[0] }}"
+        total_capacity_gi_b: "{{ objects.total_capacity_gi_b }}"
+      register: result
+
+    - name: Set object store external ID
+      ansible.builtin.set_fact:
+        object_store_ext_id: "{{ result.response.ext_id }}"
+
+    - name: Create a temporary JSON file for certificate details
+      ansible.builtin.copy:
+        mode: "0644"
+        content: |
+          {
+            "alternateIps": [
+              {
+                "ipv4": {
+                  "value": "{{ objects.public_network_reference.IPs[0] }}"
+                }
+              }
+            ],
+          }
+        dest: "/tmp/certificate_details.json"
+
+    - name: Create certificate for an object store
+      nutanix.ncp.ntnx_object_stores_certificate_v2:
+        object_store_ext_id: "{{ object_store_ext_id }}"
+        path: "/tmp/certificate_details.json"
+      register: result
+
+    - name: List all certificates for an object store
+      nutanix.ncp.ntnx_object_stores_certificate_info_v2:
+        object_store_ext_id: "{{ object_store_ext_id }}"
+      register: result
+
+    - name: Set certificate external ID
+      ansible.builtin.set_fact:
+        certificate_ext_id: "{{ result.response[0].ext_id }}"
+
+    - name: Fetch certificate details using external ID
+      nutanix.ncp.ntnx_object_stores_certificate_info_v2:
+        object_store_ext_id: "{{ object_store_ext_id }}"
+        ext_id: "{{ certificate_ext_id }}"
+      register: result
+
+    - name: List all object stores
+      nutanix.ncp.ntnx_object_stores_info_v2:
+      register: result
+
+    - name: Set object store name
+      ansible.builtin.set_fact:
+        object_store_name: "{{ result.response[0].name }}"
+
+    - name: List all object stores with filter
+      nutanix.ncp.ntnx_object_stores_info_v2:
+        filter: name eq '{{ object_store_name }}'
+      register: result
+
+    - name: List object stores with limit
+      nutanix.ncp.ntnx_object_stores_info_v2:
+        limit: 1
+        page: 0
+      register: result
+
+    - name: Fetch object store details using external ID
+      nutanix.ncp.ntnx_object_stores_info_v2:
+        ext_id: "{{ object_store_ext_id }}"
+      register: result
+# currently, delete object store is not supported since you need to delete bucket first
```

Comment:
Delete of object store is supported. Mention about extra step that is required manually. And uncomment the code. This statement is misleading

Reply:
Delete code is there in module.